### PR TITLE
Fix swoole server on request param.

### PR DESCRIPTION
**Pull request by jmjoy**
Fix https://github.com/apache/skywalking/issues/11488

### CODE REVIEW

**Review by wu-sheng — DISMISSED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ members = [
 
 [package]
 name = "skywalking-php"
-version = "0.7.0"
+version = "0.8.0-dev"
 authors = ["Apache Software Foundation", "jmjoy <jmjoy@apache.org>", "Yanlong He <heyanlong@apache.org>"]
 description = "Apache SkyWalking PHP Agent."
 edition = "2021"
```
Comment (wu-sheng) [inline Cargo.toml:27]: Do we need to update this?

Comment (jmjoy) [reply Cargo.toml:27]: Yes, we need to start a new version development.

**Comment by wu-sheng** [comment]
Please fix CI.

**Comment by jmjoy** [comment]
> Please fix CI.

The vagrant of Mac in CI is crash, I will retry it later.

**Review by wu-sheng — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**